packages/web-components/src/tablist/tablist.styles.ts

```diff
@@ -0,0 +1,214 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/index.js';
+import {
+  borderRadiusCircular,
+  colorCompoundBrandForeground1Hover,
+  colorNeutralForeground1,
+  colorNeutralForeground1Hover,
+  colorNeutralForeground2,
+  colorNeutralForegroundDisabled,
+  colorSubtleBackgroundHover,
+  colorSubtleBackgroundPressed,
+  curveDecelerateMax,
+  durationSlow,
+  fontSizeBase300,
+  fontSizeBase400,
+  lineHeightBase300,
+  lineHeightBase400,
+  spacingHorizontalMNudge,
+  spacingHorizontalSNudge,
+  spacingVerticalL,
+  spacingVerticalMNudge,
+  spacingVerticalS,
+  spacingVerticalSNudge,
+  spacingVerticalXXS,
+  strokeWidthThicker,
+} from '../theme/design-tokens.js';
+
+export const styles = css`
+  ${display('flex')}
+
+  :host {
+    --tabPaddingInline: inherit;
+    --tabPaddingBlock: inherit;
+    --tabIndicatorInsetInline: 0;
+    --tabIndicatorInsetBlock: 0;
+
+    box-sizing: border-box;
+    color: ${colorNeutralForeground2};
+    flex-direction: row;
+  }
+
+  :host([orientation='vertical']) {
+    flex-direction: column;
+  }
+
+  :host ::slotted([role='tab']) {
+    align-items: flex-start;
+  }
+
+  /* indicator animation  */
+  :host ::slotted([role='tab'][data-animate='true'])::after {
+    transition-property: transform;
+    transition-duration: ${durationSlow};
+    transition-timing-function: ${curveDecelerateMax};
+  }
+
+  :host ::slotted([role='tab'])::after {
+    height: ${strokeWidthThicker};
+    margin-block-start: auto;
+    transform-origin: left;
```

Comment:
Will this conflict with document direction?

Reply:
Logical properties don't work with `transform-origin`. I'll need to spin up another story to verify RTL.

Reply:
Verified that everything works as expected in RTL. Will have a story up for it soon. I think we actually lucked out a bit because `--tabIndicatorOffset` uses `getBoundingClientRect.x` which would also be directional-based.